<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editCourseMessage</class>
 <widget class="QWidget" name="editCourseMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>448</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>449</width>
    <height>361</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>编辑课程信息</string>
  </property>
  <widget class="QPushButton" name="noButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="yesButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>421</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>161</width>
      <height>251</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="setcourseID">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="setcourseName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="setCredit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="settime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>2022年春</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2022年秋</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2023年春</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2023年秋</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2024年春</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2024年秋</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="setteacherName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="setcapacity">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="setMajor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>经济学类</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>财政学类</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>金融学类</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>经济与贸易类</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>法学</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>中国语言文学类</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>教育学</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>中国语言文学类</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>外国语言文学类</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>历史学</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>数学类</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>物理学类</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>化学类</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>机械类</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>电子信息类</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>自动化类</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>计算机类</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>航空航天类</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="setCharator">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>通识课</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>专业必修课</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>专业限选课</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>选修课</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QLabel" name="labCourseID_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>72</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>课程号</string>
    </property>
   </widget>
   <widget class="QLabel" name="labCourseName_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>72</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>课程名</string>
    </property>
   </widget>
   <widget class="QLabel" name="labScore_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>学分</string>
    </property>
   </widget>
   <widget class="QLabel" name="labTime_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>72</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>开设学期</string>
    </property>
   </widget>
   <widget class="QLabel" name="labCharater_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>72</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>专业类别</string>
    </property>
   </widget>
   <widget class="QLabel" name="labTeacher_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>任课老师</string>
    </property>
   </widget>
   <widget class="QLabel" name="labCapacity_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>72</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>课容量</string>
    </property>
   </widget>
   <widget class="QLabel" name="labCharater_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>72</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>课程属性</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-290</x>
     <y>-290</y>
     <width>1071</width>
     <height>841</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/image/R-C.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>noButton</zorder>
  <zorder>yesButton</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
